app/src/main/java/com/hihi/twiliosms/Twilio.java

```diff
@@ -47,26 +47,24 @@ private String buildPostData(HashMap<String, List<String>> params) {
                 isFirst = false;
 
             if (key.equals(KEY_MEDIA_LIST))
-                params.get(key).forEach(data -> stringBuilder.append(KEY_MEDIA).append("=").append(encodePostData(data)));
+                for (String data : params.get(key))
```

Comment:
Could this job not be done with either  `UriBuilder` (java) or `Uri.Builder()` (android)?

Reply:
Simplifying the StringBuilder ended up being the best solution in this scenario and allowed the existing test suite to continue working without refactor.